.github/workflows/ubuntu.yaml

```
@@ -0,0 +1,24 @@
+name: CI Checks
+on: [ push, pull_request, workflow_dispatch ]
+jobs:
+  ubuntu_build:
+    runs-on: ubuntu-latest
+    container:
+      image: openquantumsafe/ci-ubuntu-focal-x86_64:latest
+    steps:
+    - uses: actions/checkout@v4
```

Nit: in `liboqs` we've started pinning dependencies for [supply chain security](https://github.com/open-quantum-safe/liboqs/issues/1706#issuecomment-2066210270), so we might as well do it here too. 
```suggestion
    - uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # pin@v4
```